typecover.go

```diff
@@ -183,6 +186,13 @@ func findType(pass *analysis.Pass, targetType string) types.Type {
 	return nil
 }
 
+func pkgMatch(path, pkgName string) bool {
+	for strings.Contains(path, "/vendor/") {
```

Comment:
Is this `for` to intentionally to strip `"/vendor/"` recursively?

Reply:
Exactly, yeah, just a lazy way to strip everything up to and including the last /vendor/. LastIndexOf would work too.